src/sentry_api.rs

```diff
@@ -23,4 +23,14 @@ impl SentryApi {
 
         response.json().await
     }
+
+    pub async fn get_latest_new_state(
+        &self,
+        validator: &ValidatorDesc,
+    ) -> Result<Vec<NewStateValidatorMessage>, Error> {
```

Comment:
In the Rust validator we've defined the return type as a `primitives::sentry::ValidatorMessageResponse`:
```rust
#[derive(Serialize, Deserialize, Debug)]
#[serde(rename_all = "camelCase")]
pub struct ValidatorMessageResponse {
    pub validator_messages: Vec<ValidatorMessage>,
}
```

Same goes for the JS impl:
https://github.com/AdExNetwork/adex-validator/blob/master/routes/channel.js#L113

You can also pass a  `?limit=1` (see `primitives::sentry::ValidatorMessagesListQuery`).